### ISSUE TITLE
Issue when orientation changes

### ISSUE THREAD

**Issue by janakaonline** [hoomazoid/CTSlidingUpPanel#15]
Hi,
First of all thank you very much for the effort you have put in to this project.

I came across this issue. When we change the orientation bottom slider is not resetting its initial screen values. Because of that, when I go from portrait to landscape, the bottom panel disappears.

How can I overcome this?

Is there a way to re init the view when orientation changes?

Thank you

**Comment by andriadze** [comment]
Right now, what I think you can do is simply recreate Bottom slide
controller when you detect orientation change. It should pick up new screen
size and reset positions on it's own afterwards.

**Comment by andriadze** [comment]
Fixed in version: 1.1.0

Use this in your view controller!
```swift 
override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
        super.viewWillTransition(to: size, with: coordinator)
        bottomController?.viewWillTransition(to: size, with: coordinator)
    }
```
